app/src/main/res/layout/activity_menu.xml

```diff
@@ -25,21 +25,21 @@
         app:layout_constraintBottom_toTopOf="@id/button_parking_automatic_release"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/button_parking_add_reservation" />
+        app:layout_constraintTop_toBottomOf="@id/button_parking_add_reservation" />
 
     <Button
         android:id="@+id/button_parking_automatic_release"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="5dp"
         android:text="@string/button_parking_automatic_release"
-        app:layout_constraintBottom_toTopOf="@id/reservation_fragment"
+        app:layout_constraintBottom_toTopOf="@id/container_fragment"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/button_parking_release" />
 
     <FrameLayout
-        android:id="@+id/reservation_fragment"
+        android:id="@+id/container_fragment"
```

Comment:
Rename to include a reference to the screen this view is in, for example: menu_fragment_container